public/index.html

```diff
@@ -52,6 +52,7 @@
     -->
 
     <title>Swapr</title>
+    <meta name="description" content="A governance-enabled automated market maker with adjustable fees." />
```

Comment:
I think we can move it as the default description to PegaMetaData, what do you think?

Reply:
Till we have Server Side rendering description is of no immeidate use since we have social media descriptions seperately.

 We need to change `og:title `  `og:description` `twitter:title` `twitter:description` also get the full benifit from social media as well.
 
 As of now I feel description can be static. And I think helmet will override the exisitng meta if its already there.